portal-web/test/functional/com/liferay/portalweb/macros/User.macro

```diff
@@ -1462,9 +1462,11 @@ definition {
 
 		if (IsElementPresent(checkboxName = "Require Password Reset", locator1 = "Checkbox#ANY_CHECKBOX")) {
 			if (${requirePasswordReset} == "true") {
-				Check(
-					checkboxName = "Require Password Reset",
-					locator1 = "Checkbox#ANY_CHECKBOX");
+				if (IsNotChecked(checkboxName = "Require Password Reset", locator1 = "Checkbox#ANY_CHECKBOX")) {
```

Comment:
@Kvale1733 its preffered to do if (A&B) instead of if (A) { if (B)} in almost every case for readability.